### PR TITLE
spice up the hosted game info

### PR DESCRIPTION
**Pull request by GrotheFAF**
add teams + observer info (behind N/M Player)
color the player+team if enough player and/or team match
add sim mods hint
niced tooltip

### CODE REVIEW

**Comment by GrotheFAF** [comment]
There might be a better solution for the highlight, but I couldn't figure one out.
Sugestions welcome.

File: src/model/game.py [hunk 1]
```diff
@@ -90,6 +90,13 @@ def __init__(self,
         self._live_replay_timer.timeout.connect(self._emit_live_replay)
         self.has_live_replay = False
 
+        # Update highlight
```
Comment (Wesmania) [inline src/model/game.py:93]: Model classes are supposed to represent games / players etc. as themselves, not tied to any UI presentation, so update signals should only be emitted if the game actually changes. I think it would be better to put this timer in gameitem, wherever the game update signals are connected to.

**Review by Wesmania — CHANGES_REQUESTED** [review]
I think it's good, qt blinking stuff seems to be usually implemented this way. It should be moved out of model classes, though.

File: src/model/game.py (hunk 1 above)
Comment (GrotheFAF) [reply src/model/game.py:93]: I tried GameItem first ... coudn't figure out where to put the rest (aside form the timer in init)
The call stack for gameitem: runFAF .exec_() -> GameItemDelegate.paint
I'm clueless here ... and I really wanne know.

Comment (Wesmania) [reply src/model/game.py:93]: Okay, I see where the difficulty is. After we switched to a typical qt model / view stuff (which does give us nifty things like perfect sorting, filtering and consistency of data), we lost what little explicit control qt gives us in case of QWidgetLists (which isn't much anyway).

The way it basically works is, we implement a Qt model of a list of things, and this model and its things (GameModelItem and GameModel) emit signals when they change (much like our own player / playerset etc. classes). When we give this to a qt view, it does all visual updates by itself, using our own delegate class to actually draw the stuff (GameItemDelegate).

Long story short, the GameModelItem class in gamemodelitem.py is the class that represents 'the UI representation of a game and all the times it changes'. We can put the timer and the 'highlight' attribute there, and emit the 'updated' signal (which in itself means 'update the UI representation').

As for showing the actual blinking, the 'data' object in GameItemDelegate (line 76 for first appearance) is exactly the GameModelItem stored in our model and passed to the delegate. We can read the 'should highlight' attribute from it in line 186.

Comment (GrotheFAF) [reply src/model/game.py:93]: Yes, I thought something was strange. As I understand it (oldschool), GameItem is 'just' a function (used here to prepare and display games in a magic fashion 'model /view stuff'  ;-)
I will try GameModellItem (even thought it works, the current solution seems wastefull to me)
(and I asume you meant L226 not L186 (wrong 'silver') I get data.highlight there, that's fine)

Comment (Wesmania) [reply src/model/game.py:93]: Yeah, pretty much. If we ever need game stuff in the UI to update based on friend stuff, or players, or something else that affects their display, GameModelItem is the place to go.

Comment (GrotheFAF) [reply src/model/game.py:93]: was an easy move to gammodelitem... worked right away .. but I made a git  mess. made a new PR
